code/game/objects/items/devices/radio/headset.dm

```diff
@@ -193,8 +193,8 @@
 	ks1type = /obj/item/device/encryptionkey/merchant
 
 /obj/item/device/radio/headset/heads/captain
```

Comment:
Change the path too, while you're at it.

Reply:
ok.